t/run/todo.t

```diff
@@ -100,6 +100,21 @@ my $is_debugging_build = $Config{cppflags} =~ /-DDEBUGGING/;
 
 our $TODO;
 
+TODO: {
+    local $::TODO = 'GH 5835';
+    my $results = fresh_perl(<<~'EOF', {});
+        $^W = 1;
+        print "$^W";
+        {
+            local $^W = $^W;
+            print "$^W";
+        }
+        print "$^W";
+        EOF
+    is($?, 0, 'perl exited normally');
+    is($results, '111', 'localized $^W behaves like ordinary localized scalar');
```

Comment:
It's not evident to me why `111` should be the expected value (2nd argument) in this unit test.

Reply:
So, the first print is responsible for the first 1.  That is the correct value.
The 2nd print is responsible for the second 1.  That is the correct value.
And the 3rd print is responsible for the third 1.  That is the correct value.

So the desired value is 111.  But it isn't.  So the test fails, but the TODO makes it expect failure, so the failure doesn't count towards the entire file failing or not.   If the result were 111, the test would unexpectedly pass, and that fact would be noted in the output